src/Compilers/CSharp/Test/Syntax/Parsing/StatementParsingTests.cs

```diff
@@ -2429,6 +2548,87 @@ public void TestUsingWithDeclarationWithMultipleVariables()
+        {
+            var text = "using T a = b, c = d;";
+            var statement = this.ParseStatement(text);
+
+            Assert.NotNull(statement);
+            Assert.Equal(SyntaxKind.LocalDeclarationStatement, statement.Kind());
+            Assert.Equal(text, statement.ToString());
+            Assert.Equal(0, statement.Errors().Length);
+
+            var us = (LocalDeclarationStatementSyntax)statement;
+            Assert.NotNull(us.UsingKeyword);
+            Assert.Equal(SyntaxKind.UsingKeyword, us.UsingKeyword.Kind());
+
+            Assert.NotNull(us.Declaration);
+            Assert.NotNull(us.Declaration.Type);
+            Assert.Equal("T", us.Declaration.Type.ToString());
+
+            Assert.Equal(2, us.Declaration.Variables.Count);
+
+            Assert.NotNull(us.Declaration.Variables[0].Identifier);
+            Assert.Equal("a", us.Declaration.Variables[0].Identifier.ToString());
+            Assert.Null(us.Declaration.Variables[0].ArgumentList);
+            Assert.NotNull(us.Declaration.Variables[0].Initializer);
+            Assert.NotNull(us.Declaration.Variables[0].Initializer.EqualsToken);
+            Assert.NotNull(us.Declaration.Variables[0].Initializer.Value);
+            Assert.Equal("b", us.Declaration.Variables[0].Initializer.Value.ToString());
+
+            Assert.NotNull(us.Declaration.Variables[1].Identifier);
+            Assert.Equal("c", us.Declaration.Variables[1].Identifier.ToString());
+            Assert.Null(us.Declaration.Variables[1].ArgumentList);
+            Assert.NotNull(us.Declaration.Variables[1].Initializer);
+            Assert.NotNull(us.Declaration.Variables[1].Initializer.EqualsToken);
+            Assert.NotNull(us.Declaration.Variables[1].Initializer.Value);
+            Assert.Equal("d", us.Declaration.Variables[1].Initializer.Value.ToString());
+        }
+
+        [Fact]
+        public void TestUsingVarWithDeclarationMultipleVariablesTree()
+        {
+            UsingStatement(@"using T a = b, c = d;");
+            N(SyntaxKind.LocalDeclarationStatement);
+            {
+                N(SyntaxKind.UsingKeyword);
+                N(SyntaxKind.VariableDeclaration);
+                {
+                    N(SyntaxKind.IdentifierName, "T");
+                    {
+                        N(SyntaxKind.IdentifierToken, "T");
+                    }
+                    N(SyntaxKind.VariableDeclarator);
+                    {
+                        N(SyntaxKind.IdentifierToken, "a");
+                        N(SyntaxKind.EqualsValueClause);
+                        {
+                            N(SyntaxKind.EqualsToken);
+                            N(SyntaxKind.IdentifierName, "b");
+                            {
+                                N(SyntaxKind.IdentifierToken, "b");
+                            }
+                        }
+                    }
+                }
+                N(SyntaxKind.CommaToken);
```

Comment:
I believe both the comma and variable declarator below are part of the variable declaration